redbot/core/data_manager.py

```diff
@@ -39,6 +41,23 @@
 config_file = config_dir / "config.json"
 
 
+def save_default_config():
```

Comment:
I think this should be named `create_temp_config`, to be clearer about what it does.